mkdocs.yml

```diff
@@ -53,16 +53,18 @@ nav:
     #- Ceramic node: run/ceramic-node.md
     #- Anchor service: run/anchor-service.md
   - StreamTypes:
-      #- Overview:
+      - StreamTypes: streamtypes/streamtypes.md
```

Comment:
Seems weird to have *StreamTypes* as a subsection of itself. Why was this changed from *Overview*?

Reply:
This is a new section so nothing was changed, but updated to use Overview instead